storage_delegation/main.tf

```diff
@@ -6,16 +13,50 @@ locals {
     "eu-de"    = "eu-de.dataplatform.cloud.ibm.com",
     "jp-tok"   = "jp-tok.dataplatform.cloud.ibm.com"
   }
-  dataplatform_ui = local.dataplatform_ui_mapping[local.location]
+  create_access_policy_kms    = !var.skip_iam_authorization_policy
```

Comment:
We can directly use `var.skip_iam_authorization_policy` for count at L25. As such no need to introduce a new local variable.

Reply:
This has been implemented